lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx.hs

```diff
@@ -132,6 +133,7 @@ module Internal.Cardano.Write.Tx
     , utxoFromTxOutsInRecentEra
 
     -- * Policy and asset identifiers
+    , Value.PolicyID (..)
     , PolicyId
```

Comment:
Hi @Anviking 

Is it intentional to have two separate exports of `PolicyID`?

Reply:
I found a better solution: https://github.com/cardano-foundation/cardano-wallet/pull/4337/commits/489b925e0194a5e5126fc86f42b2d65b5439ac56